<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ReadDataByIdentifierWidget</class>
 <widget class="QWidget" name="ReadDataByIdentifierWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1307</width>
    <height>535</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout" stretch="17,83">
     <item>
      <layout class="QVBoxLayout" name="configElementsLayout">
       <item>
        <layout class="QFormLayout" name="formLayout">
         <property name="horizontalSpacing">
          <number>10</number>
         </property>
         <property name="verticalSpacing">
          <number>6</number>
         </property>
         <item row="0" column="0">
          <widget class="QLabel" name="selectInterfaceLabel">
           <property name="text">
            <string>CAN Interface:</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QComboBox" name="selectInterface"/>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="selectedIdRangesLabel">
           <property name="text">
            <string>ID categories:</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="Lindwurm::Core::CheckableComboBox" name="selectedIdCategories"/>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="sourceAddressLabel">
           <property name="text">
            <string>Source address:</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QLineEdit" name="sourceAddress">
           <property name="text">
            <string>7E0</string>
           </property>
           <property name="placeholderText">
            <string>700</string>
           </property>
          </widget>
         </item>
         <item row="3" column="0">
          <widget class="QLabel" name="targetAddressLabel">
           <property name="text">
            <string>Target address:</string>
           </property>
          </widget>
         </item>
         <item row="3" column="1">
          <widget class="QLineEdit" name="targetAddress">
           <property name="text">
            <string>7E8</string>
           </property>
           <property name="placeholderText">
            <string>7FF</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QPushButton" name="startScanButton">
         <property name="text">
          <string>Start Scan</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="stopScanButton">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>Stop Scan</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QTableView" name="dataTable">
       <property name="font">
        <font>
         <family>Source Code Pro</family>
        </font>
       </property>
       <property name="contextMenuPolicy">
        <enum>Qt::ActionsContextMenu</enum>
       </property>
       <property name="editTriggers">
        <set>QAbstractItemView::NoEditTriggers</set>
       </property>
       <property name="showDropIndicator" stdset="0">
        <bool>false</bool>
       </property>
       <property name="selectionMode">
        <enum>QAbstractItemView::ExtendedSelection</enum>
       </property>
       <property name="selectionBehavior">
        <enum>QAbstractItemView::SelectRows</enum>
       </property>
       <property name="horizontalScrollMode">
        <enum>QAbstractItemView::ScrollPerPixel</enum>
       </property>
       <attribute name="horizontalHeaderCascadingSectionResizes">
        <bool>true</bool>
       </attribute>
       <attribute name="horizontalHeaderStretchLastSection">
        <bool>true</bool>
       </attribute>
       <attribute name="verticalHeaderVisible">
        <bool>false</bool>
       </attribute>
       <attribute name="verticalHeaderDefaultSectionSize">
        <number>23</number>
       </attribute>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QProgressBar" name="progressBar">
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>0</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>Source Code Pro</family>
      </font>
     </property>
     <property name="value">
      <number>24</number>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>Lindwurm::Core::CheckableComboBox</class>
   <extends>QComboBox</extends>
   <header>utils/checkablecombobox.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
